.github/workflows/php.yml

```diff
@@ -48,3 +48,6 @@ jobs:
 
     - name: Run code style suite
       run: vendor/bin/phpcs --standard=Magento2 Magento2/Helpers/ Magento2/Sniffs
+
+    - name: Run ESLint
+        run: ../node_modules/.bin/eslint -c eslint/.eslintrc --rulesdir eslint/rules eslint/rules
```

Comment:
Needs additional step(s) to install eslint

Reply:
done